README.md

```diff
@@ -24,50 +24,66 @@ It offers:
 
 <ul>
 <li>
-    🔧 Plug ‘n’ play composable pipelines for creating datasets for
+    🔧 Plug ‘n’ play composable pipelines for creating datasets for:
     <ul>
         <li>AI image generation model fine-tuning (Stable Diffusion, ControlNet)</li>
         <li>Large language model fine-tuning (LLaMA, Falcon)</li>
     </ul>
 </li>
 <li>
-🧱 Library of off-the-shelf reusable components for
+🧱 Off-the-shelf library of easily adaptable reusable components for:
```

Comment:
Nit, but I don't think "off-the-shelf" is used correctly here. The components are off-the-shelf, not the library :slightly_smiling_face: